sample-queries/sample-queries.json

```
@@ -2983,6 +2983,22 @@
     }
   ],
   "TeamsAppSampleQueries": [
-      
+    {
+        "id": "9d9dc771-9926-4e41-b2e9-26159bb80c85",
+        "category": "Microsoft Teams",
+        "method": "POST",
+        "humanName": "create channel",
+        "requestUrl": "/v1.0/teams/{team-id}/channels",
+        "docLink": "https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/channel_post",
```

Shouldn't we be using beta? (I could be wrong)
```suggestion
        "category": "Microsoft Teams (beta)",
        "method": "POST",
        "humanName": "create channel",
        "requestUrl": "/beta/teams/{team-id}/channels",
        "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-post?view=graph-rest-beta",
```